src/apps/js_generic/js_generic.cpp

```diff
@@ -3,7 +3,12 @@
 #include "enclave/app_interface.h"
 #include "kv/untyped_map.h"
 #include "node/rpc/user_frontend.h"
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-parameter"
+#pragma clang diagnostic ignored "-Wc99-extensions"
```

Comment:
That should not be necessary if we correctly -isystem 3rdparty, could you double check?

Reply:
You're right. The include needed updating to `#include <quickjs/quickjs.h>` 👍